alert-database/src/main/java/com/synopsys/integration/alert/database/audit/AuditEntryEntity.java

```diff
@@ -28,21 +28,31 @@
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 import javax.persistence.Lob;
 import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
 import javax.persistence.Table;
 import javax.persistence.Temporal;
 import javax.persistence.TemporalType;
 
 import org.hibernate.annotations.Type;
 
+import com.synopsys.integration.alert.database.BaseEntity;
 import com.synopsys.integration.alert.database.DatabaseEntity;
 
 @Entity
 @Table(schema = "alert", name = "audit_entries")
-public class AuditEntryEntity extends DatabaseEntity {
+public class AuditEntryEntity extends BaseEntity implements DatabaseEntity {
     public static final int STACK_TRACE_CHAR_LIMIT = 10000;
-
+    private static final long serialVersionUID = 33941320371984275L;
+    @Id
+    @GeneratedValue(generator = "alert.audit_entries_id_seq", strategy = GenerationType.SEQUENCE)
+    @SequenceGenerator(name = "alert.audit_entries_id_seq", sequenceName = "alert.audit_entries_id_seq", allocationSize = BaseEntity.DEFAULT_ALLOCATION_SIZE)
```

Comment:
I think the generator and sequence name should be different to minimize confusion because the generator represents a Hibernate object and the sequence represents a Postgres object. Adding a suffix should suffice. 
Example: 
```java 
@GeneratedValue(generator = "alert.audit_entries_id_seq_generator", ...)
@SequenceGenerator(name = "alert.audit_entries_id_seq_generator", sequenceName = "alert.audit_entries_id_seq", ...)
```